samples/setClientEndpoint.js

```diff
@@ -0,0 +1,48 @@
+// Copyright 2022 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * This application demonstrates set a custom endpoint with
+ * the Google Cloud Storage API.
+ *
+ * For more information, see the README.md under /storage and the documentation
+ * at https://cloud.google.com/storage/docs.
+ */
+
+function main(apiEndpoint = 'https://storage.googleapis.com') {
+  // [START storage_set_client_endpoint]
+  /**
+   * TODO(developer): Uncomment the following lines before running the sample.
```

Comment:
Wondering if this is unnecessary duplication since apiEndpoint is also set in the function call. But if this is the way the rest of the nodejs samples are doing it, then no worries, carry on.

Reply:
This is the way all our samples are structured. The region tag is below the function call

Reply:
Oh that makes sense, very good.